applications/tari_console_wallet/README.md

```diff
@@ -47,18 +47,37 @@ Monitoring 1 sent transactions to Broadcast stage...
 Done! All transactions monitored to Broadcast stage.
 ```
 
+- **send-one-sided**
+
+Send an amount of Tari to a public key or emoji id in a one-sided transaction.
+
+`tari_console_wallet --command send-one-sided <amount> <pubkey> <optional message>"`
+
+example:
+
+```
+$ tari_console_wallet --command "send-one-sided 1T c69fbe5f05a304eaec65d5f234a6aa258a90b8bb5b9ceffea779653667ef2108 coffee"
+
+1. send-tari 1.000000 T c69fbe5f05a304eaec65d5f234a6aa258a90b8bb5b9ceffea779653667ef2108 coffee
+
+Monitoring 1 sent transactions to Broadcast stage...
+Done! All transactions monitored to Broadcast stage.
+```
+
 - **make-it-rain**
 
 Make it rain! Send many transactions to a public key or emoji id.
 
-`tari_console_wallet --command "make-it-rain <tx/sec> <duration> <amount> <increment> <start time or now> <pubkey> <optional message>"`
+`tari_console_wallet --command "make-it-rain <tx/sec> <duration> <amount> <increment> <start time or now> <pubkey> <transaction type> <optional message>"`
+
+`<type>` can be `negotiated` or `one_sided`
```

Comment:
I prefer the term "interactive" to "negotiated" and since it's generally referred as such in discussions/issues